spec.md

```diff
@@ -99,7 +97,8 @@ The `Span` interface must have the following capabilities:
 
 The `Tracer` interface must have the following capabilities:
 
-- Start a `Span` that has no parent, commonly referred to as a *root* `Span` **(py: `start_trace`, go: `StartTrace`)**
+- Start a new child `Span` with a given operation name. With no parent, commonly referred to as a *root* `Span`, no trace attributes exist. With a parent any trace attributes must be passed through into the created child. **(py: `create_span`, go: `CreateSpan`)**
```

Comment:
Sorry, I was unclear: in the case that you do want to start traces at the edge, you would still have this problem.  Imagine a tomcat web server which receives requests from end-users of the application.  I want to start traces when they enter tomcat, only at the HTTP server level, not be sampling for starting new traces in, say, JDBC instrumentation spans.  You would not be able to use `NO_NEW_TRACES` at a `Tracer` level, because this would preclude sampling at the edge and also the interior.